lib/services/ios-debug-service.ts

```diff
@@ -212,7 +225,7 @@ export class IOSDebugService extends DebugServiceBase implements IPlatformDebugS
 				this.$logger.info("'--chrome' is the default behavior. Use --inspector to debug iOS applications using the Safari Web Inspector.");
 			}
 
-			this._socketProxy = await this.$socketProxyFactory.createWebSocketProxy(this.getSocketFactory(device));
+			this._socketProxy = await this.$socketProxyFactory.createWebSocketProxy(this.getSocketFactory(device), device.deviceInfo.identifier);
```

Comment:
device is not a mandatory argument, so you may receive `Cannot read property 'deviceInfo' of undefined.`